app/locales/es/about.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable quotes, quote-props */
+
+export default {
+    'copy': 'La aplicación de audio para unir todos los servicios de transmisión de música. Reproducir canciones ' +
```

Comment:
Could you change the indentation from 4-spaces to 2-spaces?

Reply:
ok I change de indentation..this spanish is for spain.